### PR TITLE
localize string variables

### LINKED ISSUE #854: Review that all strings in the app are properly localized for new functionality.
A lot of new functionality has gone into the app but the strings are not referenced from the localized strings file. 

Move strings from:

- ui/modal/info_modal_layouts.go
- ui/page/components/proposal_list.go
- ui/page/staking/overview.go

To: 

ui/values/strings.go

Insert english placeholders in the languages files for now for other languages.

### PR DESCRIPTION
**Pull request by Sirmorrison**
Fix #854

This PR localize string variable on the godcr app.

String localization includes Spanish and French language local which was archived using google translate.

### CODE REVIEW

File: ui/page/components/utils.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ func RetryFunc(retryAttempts int, sleepDur time.Duration, funcDesc string, errFu
 			if i > 1 {
 				sleepDur *= 2
 			}
-			fmt.Printf("waiting %s to retry function %s after error: %v\n", sleepDur, funcDesc, err)
+			log.Errorf("waiting %s to retry function %s after error: %v\n", sleepDur, funcDesc, err)
```
Comment (dreacot) [inline ui/page/components/utils.go:29]: shouldn't we localize this?

File: ui/page/governance/proposal_details_page.go [hunk 2]
```diff
@@ -523,7 +523,7 @@ func (pg *ProposalDetails) Layout(gtx C) D {
 				var err error
 				proposalDescription, err = pg.WL.MultiWallet.Politeia.FetchProposalDescription(proposal.Token)
 				if err != nil {
-					fmt.Printf("Error loading proposal description: %v", err)
+					log.Errorf("Error loading proposal description: %v", err)
```
Comment (dreacot) [inline ui/page/governance/proposal_details_page.go:526]: seems like all error formatting are intentionally left localized, can you shed more light?

File: ui/page/governance/splash_screen.go [hunk 3]
```diff
@@ -43,15 +43,7 @@ func (pg *Page) splashScreenLayout(gtx layout.Context) layout.Dimensions {
...
-							text := `<span style="text-color: gray">
-							The Decred community can participate in proposal discussions 
-							for new initiatives and request funding for these initiatives. 
-							Decred stakeholders can vote if these proposals should be approved 
-							and paid for by the Decred Treasury.
-
-							Would you like to fetch and view the proposals?
-						</span>`
-
+							text := values.StringF(values.StrGovernanceInfo, `<span style="text-color: gray">`, `<br>`, `</span>`)
```
Comment (dreacot) [inline ui/page/governance/splash_screen.go:46]: this seems like it would produce an error on the UI

File: ui/page/main_page.go [hunk 4]
```diff
@@ -577,10 +581,10 @@ func (mp *MainPage) popToFragment(pageID string) {
 	}
 }
 
-// Layout draws the page UI components into the provided layout context
+// Layout draws the page UI components into the provided C
```
Comment (dreacot) [inline ui/page/main_page.go]: i think the previous comment is still valid

File: ui/page/security_tools_page.go [hunk 5]
```diff
@@ -55,15 +55,15 @@ func (pg *SecurityToolsPage) OnNavigatedTo() {
 
 }
 
-// Layout draws the page UI components into the provided layout context
+// Layout draws the page UI components into the provided C
```
Comment (dreacot) [inline ui/page/security_tools_page.go]: i think the previous comment is still valid and more explanatory, for all instance of this

File: ui/page/staking/auto_ticket_modal.go [hunk 6]
```diff
@@ -2,7 +2,7 @@ package staking
 
 import (
 	"context"
-	"fmt"
+	// "fmt"
```
Comment (dreacot) [inline ui/page/staking/auto_ticket_modal.go]: can delete if not needed

Comment (dreacot) [inline ui/page/staking/auto_ticket_modal.go]: can delete if not needed

**Review by dreacot — CHANGES_REQUESTED** [review]

File: ui/page/governance/proposal_details_page.go (hunk 2 above)
Comment (dreacot) [reply ui/page/governance/proposal_details_page.go:526]: okay seem like we want all the logs in english, disregard

File: ui/page/components/utils.go (hunk 1 above)
Comment (dreacot) [reply ui/page/components/utils.go:29]: okay seem like we want all the logs in english, disregard

File: ui/page/governance/splash_screen.go (hunk 3 above)
Comment (dreacot) [reply ui/page/governance/splash_screen.go:46]: Screenshot from 2022-05-26 17-11-42

**Comment by dreacot** [comment]
proposals aren't showing
Screenshot from 2022-05-26 17-15-44

**Comment by dreacot** [comment]
The info icon on the governance splash page doesn't work

**Review by devchoplife — CHANGES_REQUESTED** [review]
image

**Review by dreacot — APPROVED** [review]

**Pull request merged**